third_party/badvpn/tun2socks/tun2socks.c

```diff
@@ -284,6 +284,11 @@ static void udp_send_packet_to_device (void *unused, BAddr local_addr, BAddr rem
 //==== OUTLINE ====
 #ifndef BADVPN_USE_WINAPI
 
+// Returns whether DNS resolution over UDP is supported.
```

Comment:
Cool! Let's update `third_party/badvpn/METADATA` to help track our own changes.

Reply:
Done in e23932a.